osbs/api.py

```diff
@@ -104,8 +104,17 @@ def __init__(self, openshift_configuration, build_configuration):
                             namespace=self.os_conf.get_namespace())
         self._bm = None
 
+    def _has_each_label(self, labels_needle, labels_haystack):
+        for (k, v) in labels_needle.iteritems():
+            if k not in labels_haystack:
+                return False
+            if labels_haystack[k] != v:
+                return False
+        return True
```

Comment:
This (and the labels parameter for list_builds) is removed in a later commit. Does this need to be squashed/revised?

Reply:
Yup. Let me work that out.

Reply:
Done.